<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddHost</class>
 <widget class="QDialog" name="AddHost">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Host</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Please Enter Your Information:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Name: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtFname"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Family: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtLname"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>National Code: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtNationalCode"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Personnel Code: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtPersonnelCode"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Birth Date:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dtBirthDate"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Employment Date: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dtEmploymentDate"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
